File: C++/C++Basic/04.Conditional-Statements-Exercise/07.shopping.cpp
#include <iostream>

using namespace std;

int main()
{
    double budget;
    int countVga, countCpu, countRam;

    cin >> budget >> countVga >> countCpu >> countRam;

    double priceVga = countVga * 250;
    double priceCpu = priceVga * 0.35 * countCpu;
    double priceRam = priceVga * 0.10 * countRam;

    double totalPrice = priceCpu + priceVga + priceRam;

    if (countVga > countCpu)
    {
        totalPrice *= 0.85;
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    if (budget >= totalPrice)
    {
        double moneyLeft = budget - totalPrice;

        cout << "You have " << moneyLeft << " leva left!" << endl;
    }
    else
    {
        double moneyNeeded = abs(budget - totalPrice);

        cout <<"Not enough money! You need " << moneyNeeded <<" leva more!" << endl;
    }

    return 0;
}

File: Python/Basic/04.Conditional-Statements-Exercise/07.shopping.py
budget = float(input())
number_of_vga = int(input())
number_of_cpu = int(input())
number_of_ram = int(input())

price_vga = number_of_vga * 250
price_cpu = (price_vga * 0.35) * number_of_cpu
price_ram = (price_vga * 0.10) * number_of_ram

total_sum = price_vga + price_cpu + price_ram

if number_of_vga > number_of_cpu:
    total_sum *= 0.85

if budget >= total_sum:
    print(f'You have {budget-total_sum:.2f} leva left!')
else:
    print(f'Not enough money! You need {total_sum-budget:.2f} leva more!')